# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1254799788 25200
#      Mon Oct 05 20:29:48 2009 -0700
# Node ID 01a1fd9167e05a04809aeb7d871799601af56d17
# Parent  9fba7e1da785baa3ec1eea0b14668d5184308736
 Fix typo directing users to the wrong log file

diff --git a/test/ChangeLog b/test/ChangeLog
--- a/test/ChangeLog
+++ b/test/ChangeLog
@@ -1,8 +1,12 @@
+2009-10-05  Rik  <octdev@nomad.inbox5.com>
+
+	* fntests.m: Fix typo directing users to the wrong log file
+
 2009-06-23  Robert T. Short  <octave@phaselockedsystems.com>
 
 	* @Blork/Blork.m, @Blork/bleek.m, @Blork/display.m, @Blork/get.m,
 	@Blork/set.m, @Cork/Cork.m, @Cork/click.m, @Cork/display.m,
 	@Cork/get.m, @Cork/set.m, @Dork/Dork.m, @Dork/bling.m,
 	@Dork/display.m, @Dork/gack.m, @Dork/get.m, @Dork/getStash.m,
 	@Dork/private/myStash.m, @Dork/set.m, @Gork/Gork.m, @Gork/cork.m,
 	@Gork/display.m, @Gork/gark.m, @Gork/get.m, @Gork/set.m,
diff --git a/test/fntests.m b/test/fntests.m
--- a/test/fntests.m
+++ b/test/fntests.m
@@ -225,17 +225,17 @@ try
       t2 = "failure";
     endif
     printf ("\nThere %s %d expected %s (see fntests.log for details).\n",
 	    t1, dxf, t2);
     puts ("\nExpected failures are known bugs.  Please help improve\n");
     puts ("Octave by contributing fixes for them.\n");
   endif
   if (dsk > 0)
-    printf ("\nThere were %d skipped tests (see fntest.log for details).\n", dsk);
+    printf ("\nThere were %d skipped tests (see fntests.log for details).\n", dsk);
     puts ("Skipped tests are features that are disabled in this version\n");
     puts ("of Octave as the needed libraries were not present when Octave\n");
     puts ("was built\n");
   endif
 
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".m");
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".cc");
 
